deployment/sw360/docker-entrypoint.sh

```diff
@@ -84,6 +84,34 @@ if [ "$HTTPS_HOSTS" ]; then
 fi
 
 ################################################################################
+# Setup for trusted Certificate Authorities
+#
+# $TRUSTED_CACERTS should be a comma separated list of environment variable
+# names, e.g.:
+# "TRUSTED_CA1,TRUSTED_CA2"
+# Mentioned variables ($TRUSTED_CA1 and $TRUSTED_CA2 in this example) should
+# contain the certificates, e.g.:
+# -----BEGIN CERTIFICATE-----
+# AAAAAAAAAAAAAAAAAAAAAAAAAAAA
+# -----END CERTIFICATE-----
+if [ "$TRUSTED_CACERTS" ]; then
+    for CERT_NAME in $(echo $TRUSTED_CACERTS | sed "s/,/ /g"); do
+        if [ "${!CERT_NAME}" ]; then
+          echo "Trust certificate $CERT_NAME ..."
+          while read -r line; do
+            echo $line >> $CERT_NAME.crt
+          done <<< "${!CERT_NAME}"
+          echo ${!CERT_NAME} | keytool -keystore "$JAVA_HOME/lib/security/cacerts" \
```

Comment:
What is the `echo ${!CERT_NAME} | ` good for (especially together with the `-noprompt` flag)? I think you are getting the content of the cert file via `-file $CERT_NAME.crt`.